mor/toolbox-mor/toolboxmor_main.cpp

```diff
@@ -58,71 +58,69 @@ int main( int argc, char** argv)
     heatBox->printAndSaveInfo();
     rb_model_ptrtype model = std::make_shared<rb_model_type>();
     model->setFunctionSpaces(heatBox->spaceTemperature());
+    auto rhs = heatBox->algebraicFactory()->rhs()->clone();
+    auto mat = heatBox->algebraicFactory()->matrix();
     deim_function_type assembleDEIM =
-        [heatBox](parameter_type const& mu)
+        [&heatBox,&rhs,&mat](parameter_type const& mu)
```

Comment:
@vincentchabannes is it ok to do this ? The code works but could there be any hidden behavior that I missed ?
And by the way I can not use the clone function on the matrix.